### PR TITLE
Shop rules and safety

### PR DESCRIPTION
**Pull request by JosefTaylor**
Added the tool documentation from the old shop protocols
Added shop rules as discussed in 2024-06-24 meeting

### CODE REVIEW

File: content/angle-grinder.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Angle Grinder
+
+Angle grinders are versatile tools that are also extremely dangerous. This grinder can take 4 1/2” diameter discs of different types and spins them at very high RPM. Discs you can use include abrasive cut-off wheels, flap discs (course sandpaper), and other more technical heads such as wood power carvers, wire wheels, and more..
+
+## ⚠️ Safety:
+
+- Wear safety glasses and hearing protection and a full face shield.
+- Use a mask or respirator for breathing protection whenever grinding that creates metal or wood dust.
+- Do not wear jewelry or clothing with ties or other loose dangly bits. Tie up long hair.
+- Wear the appropriate gloves to protect against sparks or wood debris.
```
Comment (gabe-ochoa) [inline content/angle-grinder.md]: It's a rotary tool so many some glove caution would also be good.

File: content/angle-grinder.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Wear long sleeves when grinding metal to protect against sparks or other flying debris.
+- When starting the grinder, hold it firmly with both hands to compensate for the rotational forces.
+- Never swing the grinder while it is on and always be aware of your surroundings and limbs when using it.
+
```
Comment (gabe-ochoa) [inline content/angle-grinder.md:15]: I'd add "never lay the grinder on its disk, specifically when still plugged into power"

File: content/cnc-router.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# The CNC Router - [X-Carve](https://shop.inventables.com/products/x-carve-1)
+
+The X-Carve CNC router works on an XY plane with limited Z-depth, capable of handling dimensions of 1000mm x 1000mm x 50mm. It is specifically designed to cut wood, and in part due to our dust collection system, should not be used for other materials.  
+Contrary to a typical CNC machine, our CNC router is designed to move relatively slowly. It relies on tool paths for rounded edges or complex shapes.
+
+## ⚠️ Safety
+
+1. Always keep the router's door closed when power is supplied to it.
+2. Secure your workpiece; if it isn't secure, it could go flying. Use a fixture or at least two clamps on each side of your piece.
```
Comment (gabe-ochoa) [inline content/cnc-router.md]: Is the tape worth a mention since it will probably be used?

File: content/cnc-router.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+3. The router collet holding the bit must be properly tightened.
+4. **Never** leave the CNC router unattended while it's running. Do not walk away, even for a bathroom break.
+5. Running a CAM program on the router is a complex process involving fixturing, homing, and execution. If you haven’t done this before, have somebody walk you through your first run.
+
```
Comment (gabe-ochoa) [inline content/cnc-router.md:24]: "Always run a test runs with the spindle off above the part to make sure you path looks correct and you have the correct offsets."

File: content/laser-cutter.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# OMTECH Laser Cutter
```
Comment (gabe-ochoa) [inline content/laser-cutter.md:1]: I can add more once this is merged and we know our set up better.

**Review by gabe-ochoa — CHANGES_REQUESTED** [review]

File: content/angle-grinder.md (hunk 1 above)
Comment (JosefTaylor) [reply content/angle-grinder.md]: I've edited a bit, though OSHA still recommends gloves for grinder use.

File: content/cnc-router.md (hunk 3 above)
Comment (JosefTaylor) [reply content/cnc-router.md]: Covered in the docs from my NYIC PR

File: content/cnc-router.md (hunk 4 above)
Comment (JosefTaylor) [reply content/cnc-router.md:24]: Covered in the docs from my NYIC PR

**Pull request merged**